### ISSUE TITLE
trace_filter errors with method handler crashed on version 2.43.0

### ISSUE THREAD

**Issue by ella-quicknode** [ledgerwatch/erigon#7536]
#### System information

Erigon version: `./erigon --version`

erigon version 2.43.0-stable-1106475e

Chain/Network: eth-mainnet

#### Expected behaviour

`erigon version 2.42.0-stable-beb97784` version worked fine 

#### Actual behaviour

In the 2.43.0 version, it responds with `method handler crashed `.  

#### Steps to reproduce the behaviour

```
  --data-raw '{
  "jsonrpc": "2.0",
  "method": "trace_filter",
  "id": 0,
  "params": [
    {
      "fromBlock": "0xc44dac",
      "toBlock": "0xc45193",
      "toAddress": [
        "0xdb22d2d37db92EA7fa6993C9f6Ead55FBb1eF4EA",
        "0x2571B97fAd74ea2073FFF07ABb0F3bfF73C95e03",
        "0x3983942D40371B264530b944f44A28C1fa1030F9",
        "0x21f73D42Eb58Ba49dDB685dc29D3bF5c0f0373CA",
        "0x4f3a120E72C76c22ae802D129F599BFDbc31cb81"
      ]
    }
  ]
}'
```

#### Backtrace

````
May 18 13:33:52 xxx erigon[23655]: [service.go:217 panic.go:884 panic.go:260 signal_unix.go:837 stream.go:184 trace_filtering.go:384 value.go:586 value.go:370 service.go:222 handler.go:494 handler.go:444 handler.go:392 handler.go:223 handler.go:316 asm_amd64.s:1598]
May 18 13:33:52 xxx erigon[23655]: [WARN] [05-18|13:33:52.179] Served                                   conn=xx.xxx.xxx.xxx:62540 method=trace_filter reqid=0 t=307.244µs err="method handler crashed"
````

**Comment by quick-pawiromitchel** [comment]
Relates to https://github.com/ledgerwatch/erigon/issues/7400

**Comment by quickchase** [comment]
https://github.com/ledgerwatch/erigon/pull/7539

**Comment by quick-pawiromitchel** [comment]
bump

**Comment by quick-pawiromitchel** [comment]
bump

**Comment by quick-pawiromitchel** [comment]
This issue is solved in v2.43, good to close ✅
